quantgov/estimator/structures.py

```diff
@@ -85,3 +113,41 @@ class CandidateModel(
             parameter values to test as values
     """
     pass
+
+
+class QGLdaModel(BaseEstimator, TransformerMixin):
+    @check_gensim
+    @check_spacy
+    def __init__(self, word_regex=r'\b[A-z]{2,}\b', stop_words=STOP_WORDS):
```

Comment:
I would think the options for `stop_words` should be:

- `None` (default): No stop words
- `True`: use built-in stop words
- A sequence: user-specified stop words.

Reply:
Not having any stop words seems to output a pretty unusable model - my thinking is it's best to have some default, and if the user chooses to override that default with None they can, but the defaults should be able to produce something usable - we could include some output if they don't provide any (e.g. "INFO: No stop words provided, using sklearn builtins"), and potentially a warning if None is passed